pages/index.jsx

```diff
@@ -15,15 +20,21 @@ export default function Home() {
         <title>PuMP | Home</title>
       </Head>
       <PageLayout>
-        <TopSection />
-        <WhatsHappening />
-        <Video
-          embedId="f5jsJWl5WS0"
-          titleText="Who Are We?"
-          descriptionText="Founded in March 2018, Prospective Medical Professionals or PuMP for short is a Canadian charity founded by youth, led by youth. Our objective is to help aspiring life and health sciences students better prepare for a future in the biology by connecting them to real life experiences."
-        />
-        <ImpactSection />
-        <ContactSection />
+				<SectionWrapper
```

Comment:
The SectionWrapper should only wrap one component and not multiple

Reply:
A `background-image` can't extend outside of a `div`'s (or other component's) border. As a result, the current `TopSection` blobby gets cut off because of this when you only wrap `TopSection`.

Ultimately it'll probably be a good idea to modify the graphic to be shorter then it is now... for now, though, I'll just remove `SectionWrapper` from the homepage until we sort something out.

Reply:
Yep, I realized that so I think we need to contact design to change the blob. Rachel Chen is the one who made this page so it would be best to contact her. And sounds good.